src/interfaces/defiAdapters/contracts/IAdapterInvestLimit.sol

```diff
@@ -3,7 +3,9 @@
 pragma solidity >=0.6.0 <= 0.9.0;
 pragma experimental ABIEncoderV2;
 
-import { DataTypes } from "../../../libraries/types/DataTypes.sol";
+
+/** @notice Named Constants for defining max exposure state */
+enum MaxExposure { Number, Pct }
```

Comment:
Now when you import this Adapter in any file
use `import "./IAdapterInvestLimit.sol"`
and not
`import {IAdapterInvestLimit}  from "./IAdapterInvestLimit.sol"`

because it will throw compile error as `MaxExposure` won't be accessible

Reply:
Fixed one import statement accordingly.